src/simplifier/AddUsingsRewriter.cs

```diff
@@ -36,7 +36,7 @@ public override SyntaxNode VisitNamespaceDeclaration(NamespaceDeclarationSyntax
             );
             
 
-            if(node.Name!=null)
+            if(node.Name!=null && node.Name.GetType().Name=="QualifiedNameSyntax")
```

Comment:
what about `node.Name?.GetType() == typeof(QualifiedNameSyntax)`? Little cleaner, no magic string constants involved :)

Reply:
Huh, why did I not think of that. I might as well clean up some more code then!